### PR TITLE
[CDAP-6772] [CDAP-6770] Handle Unauthorized to any Namespace

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-ui/app/features/home/unauthorized-namespace.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="jumbotron text-center">
+  <h1><span class="text-danger">uh oh!</span></h1>
+  <p>You do not have authorization to view any namespaces. Please contact your system administrator to get access to a namespace.</p>
```
Comment (anwar6953) [inline cdap-ui/app/features/home/unauthorized-namespace.html]: Isn't it more like - unauthorized to view a single/particular namespace?
This message says not authorized to view _any_ namespaces.

Comment (anwar6953) [inline cdap-ui/app/features/home/unauthorized-namespace.html]: Oh; nevermind.  I see now that this page is only used when no namespaces are returned by the backend.

File: cdap-ui/app/features/home/home-ctrl.js [hunk 2]
```diff
@@ -41,6 +41,9 @@ angular.module(PKG.name + '.feature.home')
           if (checkNamespace('default')){
             $state.go($state.current, {namespace: 'default'}, {reload: true});
             return;
+          } else {
+            $state.go($state.current, { namespace: rNsList[0].name }, { reload: true });
+            return;
           }
 
           // evoke backend is down
```
Comment (ajainarayanan) [inline cdap-ui/app/features/home/home-ctrl.js]: We would we reach line 50? Seems like we won't hit backend down scenario now?

File: cdap-ui/app/features/login/login.html [hunk 3]
```diff
@@ -56,8 +56,11 @@
   <div class="form-group">
     <div class="col-sm-offset-2 col-sm-10">
       <button type="submit" class="btn btn-primary" ng-disabled="submitting || loginForm.$invalid">
-        Submit
+        Login
       </button>
+      <span class="text-danger">
+        {{ error }}
```
Comment (ajainarayanan) [inline cdap-ui/app/features/login/login.html]: nit: Can we use `ng-bind`. In the worstcase something breaks or network is super slow we don't want to end up seeing `{{error}}` in UI. ng-bind prevents (this is from the instances bug we had in workers view).

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by bdmogal** [comment]
The behavior is good. Perhaps we should allow users to create a namespace from the "Unauthorized" page, if the user has the required access.

Scenario:
- I am a CDAP/Sentry Admin. 
- I've just setup CDAP and enabled authorization as well.
- I have not yet granted access to any user to any namespace.
- However, I myself have the required privileges (ADMIN on the CDAP instance) to create a namespace.
- Instead of first doing this through the CLI, I should be able to do it right from the "Unauthorized" Page.
- If a non-admin user presses this "button" to create a namespace, then he'll get the red ribbon with the unauthorized error message received from the backend.

What do you think? @nitinmotgi @ToddGreenstein

**Comment by bdmogal** [comment]
LGTM

**Pull request merged**